pkgs/servers/roon-bridge/default.nix

```diff
@@ -56,10 +57,6 @@ stdenv.mkDerivation {
       mkdir -p $out
       mv * $out
 
-      rm $out/check.sh
-      rm $out/start.sh
-      rm $out/VERSION
```

Comment:
These get removed for a reason, the issue though is the NixOS module references `start.sh` when it should just call the `RoonBridge` binary directly.

IOW, you need to fix this: https://github.com/NixOS/nixpkgs/blob/45b92369d6fafcf9e462789e98fbc735f23b5f64/nixos/modules/services/audio/roon-bridge.nix#L45

You can see how the `roon-server` module does the right thing here: https://github.com/NixOS/nixpkgs/blob/45b92369d6fafcf9e462789e98fbc735f23b5f64/nixos/modules/services/audio/roon-server.nix#L45

Reply:
thanks for the input I will get on that when I get home and open another pull request with a fixed setup